server/setup-apache.sh

```diff
@@ -1,5 +1,4 @@
 # change server www root folder from '/opt/bitnamic/apache/htdocs' to '~/flarum/public'
-grep -rl '/opt/bitnami/apache/htdocs' /opt/bitnami/apache/conf/ | xargs sed -i 's|/opt/bitnami/apache/htdocs|~/flarum/public|g'
-
+grep -rl '/opt/bitnami/apache/htdocs' /opt/bitnami/apache/conf/bitnami/ | xargs sudo sed -i  's|/opt/bitnami/apache/htdocs|/home/bitnami/flarum/public|g'
```

Comment:
I think I have just added to try it out, rolling out to the old line, the only update will be the sudo for sed because the sed using a temporary file to write and need permission